### ISSUE TITLE
[netcore][20.qna-with-appinsights] Readme uses a deleted '.bot' file

### ISSUE THREAD

**Issue by enzocanoo** [Microsoft/BotBuilder-Samples#690]
**Is your feature request related to a problem? Please describe.**
The readme has a step to update the `BotConfiguration.bot` that was recently deleted from this sample with #685.
1. Follow the [Prerequisites](https://github.com/Microsoft/BotBuilder-Samples/tree/master/samples/csharp_dotnetcore/20.qna-with-appinsights#prerequisites) from the readme.
2. The 3rd step uses a missing `BotConfiguration.bot`.

**Describe the solution you'd like**
This sample should have a dummy `.bot` file with the minimum placeholders to run the sample. For example:
```json
{
  "name": "qnamaker",
  "services": [
    {
      "type": "endpoint",
      "id": "1",
      "name": "development",
      "appId": "<YOUR_APPID>",
      "appPassword": "<YOUR_APPPASSWORD>",
      "endpoint": "http://localhost:3978/api/messages"
    },
    {
      "type": "qna",
      "id": "2",
      "name": "QnABot",
      "kbId": "<YOUR_KBID>",
      "subscriptionKey": "<YOUR_SUBSCRIPTIONKEY>",
      "endpointKey": "<YOUR_ENDPOINTKEY>",
      "hostname": "<YOUR_HOSTNAME>"
    },
    {
      "type": "appInsights",
      "id": "4",
      "name": "AppInsights",
      "tenantId": "",
      "subscriptionId": "",
      "resourceGroup": "",
      "serviceName": "",
      "instrumentationKey": "<YOUR_INSTRUMENTATIONKEY>",
      "applicationId": "",
      "apiKeys": {}
    }
  ],
  "padlock": "",
  "version": "2.0"
}
```
Using this approach, the user just needs to create the required resources and modify the `.bot` file to run the sample.

**Describe alternatives you've considered**
Another approach could be to update the readme to make **mandatory** the steps related to _Deploy this bot to Azure_.

## Additional context
Other samples have the same issue regarding the missing `.bot` file.

[enhancement]

**Comment by harrison-broadbent** [comment]
I would be happy to work on this.  Do you think that going through the examples and adding this dummy file and then correctly linking it with the examples readme is a suitable fix?

**Comment by daveta** [comment]
Hi @enzocanoo / @harrison-broadbent,
For this issue, here's the plan:
- @vishwacsena is going to create 2 other .md files for the following topics:
   - LUIS configuration using msbot and configuring using portal.
   - Application Insights configuration using msbot and configuring using portal.
   - (There will be an additional one for QnA but not relevant for this issue)

Also, within the README we will add a template for this sample, with placeholders of where you need to replace keys/connection strings/etc.

This should land later this week/next week.

**Comment by enzocanoo** [comment]
Hi @daveta, looks good to me!

Just a little comment: other samples have the same issue related to the missing `BotConfiguration.bot`. Each sample will have a different `BotConfiguration.bot` template file according to their need, right?

I think that [this readme (Set up LUIS region)](https://github.com/Microsoft/BotBuilder-Samples/tree/master/samples/javascript_nodejs/12.nlp-with-luis#set-up-luis) can be used as an example of how to change values in the `.bot` file.

**Comment by daveta** [comment]
Yes, we're in the process of rewriting the Readme's so there's a starting point for each.

**Comment by JasonSowers** [comment]
this should be addressed in #828 please provide any feedback on the changes you have
